Features/Standard/ContentTypeIcon.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Content Type icons
+
+    @javascript
+    Scenario: Content items of a type without an icon get the default icon
+        Given a custom Content Type exists
+          And a Content item of this type exists
+         When I view this Content
+         Then I see the "generic" Content Type icon attached to the title
+
+    @javascript
+    Scenario: Content items of a type with an icon get this icon
+        Given a Content item of type "folder" exists
+         When I view this Content
+         Then I see the "folder" Content Type icon attached to the title
+
+    @javascript
+    Scenario: The Content Type icon can be customized for a given Content Type with a CSS rule
+        Given a Content item of type "folder" exists
+         When I view this Content
+         Then the title element has the class "ez-contenttype-icon-folder"
+
+    @javascript
+    Scenario: Content items in the Content Tree have an icon based on the Content Type
+        Given I display the Content Tree
+         Then I see an icon based on the Content Type for each Content item
+
+    @javascript
+    Scenario: Content items in the subitem grid view without a filled image field have an icon based on the Content Type
```

Comment:
I would not specify again what icon is rendered. What matters here is that the grid item uses the icon if there is no image. What icon is used is covered by other scenarii.